jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/VolumePermissionEnricher.java

```diff
@@ -112,7 +113,7 @@ private Container createPvInitContainer(PodSpec podSpec) {
                 Map<String, String> mountPoints = extractMountPoints(podSpec);
                 return new ContainerBuilder()
                         .withName(ENRICHER_NAME)
-                        .withImage("busybox")
+                        .withImage(getConfig(Config.IMAGE_NAME,Config.IMAGE_NAME.getDefaultValue()))
```

Comment:
This is not necessary, the default value is retrieved from the config key automatically:
```suggestion
                        .withImage(getConfig(Config.IMAGE_NAME))
```

Reply:
ok noted

Reply:
> When this will be merged into master?



I understand it's your first time working with some open source project but please have some patience. We are going to merge it as soon as it's ready to merge. Right now you need to update this patch as per the changes requested by Marc.